date/authors.yml

```diff
@@ -0,0 +1,4 @@
+---
+- !ruby/object:Author
+  name: John Doe
```

Comment:
let's remove all the yml files from 'data' folder:
1) remove all the content of .gitignore
2) make commit
3) remove all the yaml files
4) make new commit
5) bring back the previous content of .gitignore
6) make commit

done